pkg/kubelet/util/csr/csr.go

```diff
@@ -39,7 +49,7 @@ import (
 func RequestNodeCertificate(client certificatesclient.CertificateSigningRequestInterface, privateKeyData []byte, nodeName types.NodeName) (certData []byte, err error) {
```

Comment:
This function is not being used for the cert rotation. The code path is slightly different as rotation starts from a template rather than constructing the attributes of the CSR.